spec/controllers/idv/image_uploads_controller_spec.rb

```diff
@@ -1,8 +1,13 @@
 require 'rails_helper'
 
+UUID_REGEX = /^[a-f0-9]{8}-([a-f0-9]{4}-){3}[a-f0-9]{12}$/
+BASE64_REGEX = /^[a-z0-9+\/]+=*$/i
```

Comment:
these constants are in the global spec so they'll leak into other files. 

I'd consider defining them as locals (`uuid_regex`, `base64_regex`) or as memoized `let` helpers (`let(:uuid_regex) { ... } `)